### PR TITLE
2.11.7 released

### PR DESCRIPTION
**Pull request by SethTisue**
not ready for merge yet. missing pieces:

* update download sizes (in index.md and the new file in _posts) after we build bundles
* add release notes to the new news post, once they're done

### CODE REVIEW

**Comment by SethTisue** [comment]
review by @adriaanm

File: download/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 title: Download
-release_version: 2.11.6
-release_date: "March 5, 2015"
+release_version: 2.11.7
+release_date: "June 23, 2015"
 other_releases: [
   ["maintenance_version", "Last 2.9.x maintenance release", 2.9.3, "February 28, 2013"],
   ["maintenance_version", "Last 2.10.x maintenance release", 2.10.5, "March 5, 2015"],
-  ["development_version", "Current 2.11.x release", 2.11.6, "March 5, 2015"],
+  ["development_version", "Current 2.11.x release", 2.11.7, June 23, 2015"],
```
Comment (sjrd) [inline download/index.md]: A `"` disappeared here.

**Comment by sjrd** [comment]
That's all from me at this point.

File: download/_posts/2015-06-23-2.11.7.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Scala 2.11.7
+start: 05 March 2015
+layout: downloadpage
+release_version: 2.11.7
+release_date: "March 05, 2015"
```
Comment (adriaanm) [inline download/_posts/2015-06-23-2.11.7.md]: date looks a bit off  :)

File: download/_posts/2007-05-21-2.5.0.final.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ show_resources: "true"
 permalink: /download/2.5.0.final.html
 requirements: "This Scala software distribution can be installed on any Unix-like or Windows system. It requires the Java runtime version 1.6 or later, which can be downloaded <a href='http://www.java.com/'>here</a>."
 resources: [
-  ["-main-unixsys", "scala-2.5.0-final.tar.gz",             "/files/archive/scala-2.5.0-final.tar.gz",                "Max OS X, Unix, Cygwin",  "12 MB"],
+  ["-main-unixsys", "scala-2.5.0-final.tar.gz",             "/files/archive/scala-2.5.0-final.tar.gz",                "Mac OS X, Unix, Cygwin",  "12 MB"],
```
Comment (adriaanm) [inline download/_posts/2007-05-21-2.5.0.final.md]: Max OS C!

**Comment by SethTisue** [comment]
fixes made, thanks

**Comment by SethTisue** [comment]
now ready to be merged... but not until we're ready to announce, of course!

**Pull request closed without merging**